locations/mlk-nk.yml

```diff
@@ -129,22 +117,11 @@ networks:
       mlk-nk-core: 1
       mlk-nk-switch: 2
       mlk-nk-rhnk: 3
-      mlk-nk-zwingli: 4
-      mlk-nk-emma: 5
-      mlk-nk-nf-wbp1: 6
+      mlk-nk-nno-5ghz: 4
+      mlk-nk-so-5ghz: 5
       mlk-nk-nf-wbp2: 7
       mlk-nk-nf-wbp3: 8
 
-location__channel_assignments_11a_standard__to_merge:
-  # Channel - Bandwidth - TX Power
-  mlk-nk-rhnk: 36-20
-
-location__channel_assignments_11g_standard__to_merge:
-  # Channel - Bandwidth - TX Power
-  mlk-nk-nf-wbp1: 1-20
-  mlk-nk-nf-wbp2: 6-20
-  mlk-nk-nf-wbp3: 11-20
```

Comment:
Why did you removed the channel assignments for 2,4GHz?

Reply:
Because I checked the devices and they were set to the default (13-20). The configuration was not matching the setup here.